packages/components/src/FilterBar/context/reducer/setupFiltersState.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { Filters } from "../../types"
+import { baseFilterAttributes } from "./__testdata"
+import { setupFiltersState } from "./setupFiltersState"
+
+type Values = {
+  flavour: string
+  sugarLevel: number
+}
+
+const filters = [
+  { id: "flavour", name: "Flavour", Component: <div /> },
+  {
+    id: "sugarLevel",
+    name: "Sugar Level",
+    Component: <div />,
+    isRemovable: true,
+  },
+] satisfies Filters<Values>
+
+describe("setupFiltersState", () => {
```

Comment:
Don't think that is yet written down anywhere, but when I last went and cleaned up tests, I tried to establish a convention where the main `describe` block follows this pattern:
- When a component is tested it says `"<Component />"`
- When it's a function it indicates so with `"myFunction()"`

Reply:
Actually noticed you did this later on :)

Reply:
Ah, must have missed it! Good spot!